pkg/collect/command_run.go

```diff
@@ -71,7 +71,7 @@ func createArchive(dir string, filenames []string, buf io.Writer) error {
 }
 
 func addFileToArchive(tw *tar.Writer, filename string) error {
-	file, err := os.Open(filename)
+	file, err := os.Open(filename) // #nosec G304
```

Comment:
Using `filepath.Clean` should be enough to silence `G304`.

Reply:
Thanks for the hint! Addressed with https://github.com/gardener/network-problem-detector/pull/77